docs/references/files/fossa-yml.md

```diff
@@ -105,15 +106,24 @@ The project fields allow you to configure settings for the project you are inter
 
 > Note: `name`, `team`, `policy`, `link`, and `jiraProjectKey` can only be set when creating a project (running `fossa analyze` for the first time).  Otherwise, they will be silently ignored (we would like to make this a visible warning in the future).
 
+#### `project.locator:`
+The project Locator defines a unique ID that the FOSSA API will use to reference this project within FOSSA. The project locator can be found in the UI on the project `Settings` page listed as the `Project Locator` underneath the `Project Title` setting.
+
+<img src="../../assets/project-locator-example.png">
+
 #### `project.id:`
-The project ID defines a unique ID that the FOSSA API will use to reference this project. The project ID can be found in the UI on the project settings page listed as the "Project Locator" underneath the "Project Title" setting. For example, if the "Project Locator" value of `custom+1/foo` is provided in the FOSSA UI, use `foo` for the `project.id`.
+The project ID defines a unique ID that the FOSSA API will use to reference this project within your organization. The project ID is a specific portion of the project locator and can be found in the UI on the project `Settings` page listed as the "Project Locator" underneath the "Project Title" setting. For example, if the "Project Locator" value of `custom+1/foo` is provided in the FOSSA UI, use `foo` for the `project.id`.
```

Comment:
1. project.locator only applies for commands `project` and `release-group` so if `project.id` is supplied, it just disregards the value from `project.id`.
2. Not necessary to supply both as they're used for different commands. 
`project.locator` -> `fossa project` & `fossa release-group`
`project.id` -> `fossa analyze` uses the `project.id` to target specific project revisions
3. There are complications with extracting the id from the project.locator. When running `fossa analyze` on a new project,  users are able to set a custom `project.id`. Removing `project.id` means that users lose this functionality, though I am not sure as to how often this feature is used.

Reply:
1. Why is it necessary to use a project locator for edit? Based on the description it sounds like a project.id would suffice since a user's API key implies an organization. I'm mainly trying to figure out if we absolutely need these both. They're two values that are very similar so I think it would be easy for a user to use the wrong one. 
2. The outcome of discussion on (1) aside, I think that this should be spelled out where the values are defined.
3. It sounds like what you're saying is that extracting it from a project locator is possible, but it would not be backwards compatible. Is that correct? I think you're right regardless though. Plus, it's a crummy UX to force someone to put their org ID before their project name. 

Am I mistaken for thinking that the only difference between the locator and id is that one has the org specifier?

Reply:
In most cases, the only difference between locator and id is the org specifier.

Given the case that we have:
`project.id` -> example
`orgId` -> 1 

Usually we get:

`project.locator` -> custom+1/example 

^ But it isn't always the case that project locators are structured like this (confirmed with core)

In this case we can build up the project locator with just `project.id` and we're able to edit this project accordingly. 

However, there are some instances where we normalize the project locator. Sometimes we can have:
`project.locator` -> git+git/example (or something like this)

In this case there would be no way for us to edit this specific project with just `project.id` from the cli.

Reply:
Projects uploaded through the CLI are structured:

`custom+23964/test-package$<version>`

reference: https://docs.google.com/document/d/1IZ7TAfsgQekfeq-2wJVtZplp4CquPCzrq4uZlB3OBDY/edit#heading=h.25qf6k4027ah

So if we only want to limit editing projects to only projects uploaded through the CLI, then we can get rid of `project.locator` and just use `project.id`. 

Thoughts?

Reply:
We had a discussion out-of-band and these are the notes:
Notes:
* Discussed how you could have a scenario where project info is set by `analyze` but that same block is unusable without adding a `project-locator` key for project edit.
  * In the current PR `project edit` and `release-group...` require `project.locator` but `fossa analyze` uses `project.id`
     * Unreasonable to expect a user to know what the project.locator will be for a yet-to-exist project, so we need to keep project.id
     * To properly update projects in core we need to send a full locator
       * Analyze gets around this by simply assuming that the locator is something like custom+XXXX/<project.id>
       * Solution(s):
          * When project edit sees only project.id , do the same thing as analyze to generate a locator.
          * Also: either forbid both project.id and project.locator from being set OR prefer project.locator to project.id.
          * For consistency, analyze needs to be set up to pass project.locator through I think. No matter the reason, all commands should select the same project given the same configuration.
* Because organizations can require a `--policy` on project creation we can't begin deprecating the `fossa analyze` project flags until we have a `fossa project create` command.
* From after our talk: I think that project edit should also accept policy-id if it doesn't. See the analyze flags.